client/post-editor/editor-post-formats/index.jsx

```diff
@@ -77,18 +90,20 @@ const EditorPostFormats = React.createClass( {
 	},
 
 	renderPostFormats: function() {
+		var selectedFormat = this.getSelectedPostFormat();
+
 		return this.getPostFormats().map( function( postFormat ) {
 			return (
 				<li key={ postFormat.slug } className="editor-post-formats__format">
 					<label>
 						<FormRadio
 							name="format"
 							value={ postFormat.slug }
-							checked={ postFormat.slug === this.props.value }
+							checked={ postFormat.slug === selectedFormat }
 							onChange={ this.onChange } />
 						<span className="editor-post-formats__format-label">
 							<span className={ 'editor-post-formats__format-icon' } >
-								<Gridicon icon={ this.getPostFormatIcon( postFormat ) } size={ 20 } />
+								<Gridicon icon={ this.getPostFormatIcon( postFormat ) } size={ 18 } />
```

Comment:
>There was a lint failure on this line

If we don't want to concern ourselves with getting designer sign-off on the adjusted size, we can specify the `nonStandardSize` prop to `<Gridicon />` to bypass the warning ([see `bin/check-gridicon-format`](https://github.com/Automattic/wp-calypso/blob/master/bin/check-gridicon-format)).